bin/build_utils

```diff
@@ -25,3 +25,25 @@ function gen_versions() {
     echo $version
   done
 }
+function git_tag_suffix() {
+  if  is_tagged_commit
+  then
+    echo "-$(git_tag)"
+  else
+    echo ""
+  fi
+}
+function is_tagged_commit() {
```

Comment:
please add an empty line before this function

Reply:
done